package.json

```diff
@@ -216,6 +216,16 @@
           "orders.item.put"
         ]
       },
+      {
+        "permissionName": "ui-orders.order.unopen",
+        "displayName": "Orders: Unopen purchase orders",
+        "description": "",
+        "visible": true,
+        "subPermissions": [
+          "orders.item.put",
+          "orders.item.unopen"
```

Comment:
@craigmcnally could you please check if it fits new permissions set guidelines? thanks